test/test_core.py

```diff
@@ -5757,14 +5757,22 @@ def test_fs_nodefs_nofollow(self):
     self.emcc_args += ['-lnodefs.js']
     self.do_runf('fs/test_nodefs_nofollow.c', 'success')
 
+  @crossplatform
   @requires_node
   def test_fs_nodefs_readdir(self):
     # externally setup an existing folder structure: existing/a
     if self.get_setting('WASMFS'):
       self.set_setting('FORCE_FILESYSTEM')
+    if not WINDOWS and not MACOS:
+      # Add an entry that isn't a directory, file, or link to test that we handle
+      # it correctly.
+      os.mkfifo(os.path.join(self.working_dir, 'named_pipe'))
     os.makedirs(os.path.join(self.working_dir, 'existing', 'a'))
     self.emcc_args += ['-lnodefs.js']
-    self.do_runf('fs/test_nodefs_readdir.c', 'success')
+    suffix = ""
+    if self.get_setting('WASMFS'):
+      suffix = ".wasm"
```

Comment:
maybe `.wasmfs`?   `.wasm` is make is it looks like we are talking about wasm vs js

Reply:
Use single quotes here.